package.json

```diff
@@ -7,7 +7,7 @@
     "scripts": {
```

Comment:
Please rebase with master and also drop the changes on `sandbox/rd3g.sandbox.**`